src/Sri.php

```diff
@@ -17,21 +18,21 @@ public function __construct(string $algorithm)
             : 'sha256';
     }
 
-    public function html(string $path, bool $useCredentials = false): string
+    public function html(string $path, bool $useCredentials = false): HtmlString
     {
         if (! config('subresource-integrity.enabled')) {
-            return '';
+            return new HtmlString('');
         }
 
         try {
             $integrity = $this->hash($path);
         } catch (\Exception $e) {
-            return '';
+            return new HtmlString('');
         }
 
         $crossOrigin = $useCredentials ? 'use-credentials' : 'anonymous';
 
-        return "integrity='{$integrity}' crossorigin='{$crossOrigin}'";
+        return new HtmlString("integrity=\"{$integrity}\" crossorigin=\"{$crossOrigin}\"");
```

Comment:
I would swap the quotes to make it more readable so that we don't have to escape the double quotes.

```suggestion
        return new HtmlString('integrity="' . $integrity . '" crossorigin="' . $crossOrigin . '"');
```